platformio.ini

```diff
@@ -12,7 +12,7 @@
 ; default_envs = travis_esp8266, travis_esp32
 
 # Release binaries
-default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth, esp32s2_saola, esp32c3
+; default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth, esp32s2_saola, esp32c3
```

Comment:
remove updates to platforio.ini from your PR.

use platformio_override.ini for your custom build options

Reply:
fixed.